core/amber/src/main/scala/edu/uci/ics/amber/engine/common/promise/PromiseCompleted.scala

```diff
@@ -0,0 +1,4 @@
+package edu.uci.ics.amber.engine.common.promise
+
+// the "void" return value of a promise.
+case class PromiseCompleted()
```

Comment:
Should we have an object `ReturnPayload` too where all case class like PromiseCompleted, ExecutionPaused are put?

Reply:
We already have that.